artemis-server/src/test/java/org/apache/activemq/artemis/tests/util/ActiveMQTestBase.java

```diff
@@ -251,6 +251,9 @@ public void runAfter() {
    }
 
 
+   protected void clearServers() {
```

Comment:
This doesn't appear to be used anywhere. Was it added by mistake?

Reply:
Believe its used in the new test which subclasses this (I also thought it was unused initially)

Reply:
although I will make the other call to servers.clear to use this method explicitly ...